dev/diagrams/on-app-booting-hook.md

```diff
@@ -0,0 +1,43 @@
+# On App Booting Hook
+
+<!-- TOC -->
+
+- [Use Is App Booted Hook](#use-is-app-booted-hook)
+  - [Overview](#overview)
+
+<!-- /TOC -->
+
+## Overview
+This is a basic activity diagram for the useIsAppBootedHook.
+
+
+```plantuml
+
+@startuml
+
+title useIsAppBootedHook()
+
+participant "End User" as user
+
+box "FE" #Lightblue
+  participant "Component" as fe
+  participant "Hook: useIsAppBooted" as hook
+box
+
+activate user
+  user -> fe: open app
+  activate fe
+    fe -> hook: useIsAppBooted
+    activate hook
+      hook <- hook: handleSignOutApp()
+      hook <- hook: router.push(DEFAULT_MAIN_APP_PAGE)
```

Comment:
```suggestion
      hook -> hook: handleSignOutApp()
      hook -> hook: router.push(DEFAULT_MAIN_APP_PAGE)
```

Better to make it `a -> a`

Good Example:
![image](https://github.com/ajktown/docs/assets/53258958/32ff1c64-35b2-4da8-9ebd-0cd9d1aed320)

Bad example:
![image](https://github.com/ajktown/docs/assets/53258958/c0897574-599b-4625-8fa8-a478a0c03620)